examples/Titanic/test_data/titanic_survived.csv

```diff
@@ -0,0 +1,1310 @@
+age,sibsp,parch,fare,female,male,embark_C,embark_Q,embark_S,class_1,class_2,class_3
+29.0,0,0,211.3375,1,0,0,0,1,1,0,0
+0.92,1,2,151.55,0,1,0,0,1,1,0,0
```

Comment:
Lets keep first 5 or 10 rows . Its just a sample input to run the file. Please add the link to dataset in the README.md file if in case user wants to access it.